pandas/tests/groupby/test_categorical.py

```diff
@@ -67,6 +68,12 @@ def setUp(self):
              'E': np.random.randn(11),
              'F': np.random.randn(11)})
 
+    def test_level_groupby_get_group(self):
+        # gh15155
+        testGroupBy = self.mframe.groupby(level=["first"])
+        assert_numpy_array_equal(testGroupBy.get_group("foo").values,
```

Comment:
don't use ``assert_numpy_array_equal``, rather directly test the ``get_group`` and assert the results